examples/cli_example_no_option.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DoMapping.
+# Copyright (C) 2015, 2016 CERN.
```

Comment:
The project name is DoSchema, not DoMapping.
This file is created in 2016, no need to add 2015 in the copyright.